src/cute_app.cpp

```diff
@@ -254,7 +254,7 @@ CF_Result cf_make_app(const char* window_title, CF_DisplayID display_id, int x,
 		} else if (use_vulkan) {
 			device_name = "Vulkan";
 		}
-		device = SDL_GpuCreateDevice(SDL_ShaderCross_GetShaderFormats(), true, false, device_name);
+		device = SDL_CreateGPUDevice(SDL_ShaderCross_GetShaderFormats(), false, device_name);
```

Comment:
This function loses a boolean, but, I couldn't find what the boolean was doing previously so I just omit it. The samples seemed to run fine without it.